README.md

```diff
@@ -166,9 +166,24 @@ spec:
 | Private Builder Image Registry     | ⚪️     |      |    |
 | Runtime Base Image    | ⚪️    |      |    |
 | Binary builds         |       |      |    |
+| Image pull policy     |       |      |    |
+| Inject source via config map         |       |      |    |
+| Inject source via secret         |       |      |    |
+| Inject source via image         |       |      |    |
+| Set image label/ens   |       |      |    |
+| Triggers (SCMs)       |       |      |    |
```

Comment:
https://github.com/tektoncd/triggers already provides this, with more features already IMO, than what we have in buildv1

So I would envision this being something buildv2 does solely by leveraging tekton

Reply:
opened https://github.com/redhat-developer/build/issues/51